gradle/node.gradle

```diff
@@ -16,7 +16,7 @@
  */
 
 configure([project(":solr:packaging"), project(":solr:solr-ref-guide"), project(":solr:webapp")]) {
-    apply plugin: "com.github.node-gradle.node"
+    apply plugin: libs.plugins.nodegradle.node.get().pluginId
```

Comment:
TBH I liked it as it was -- a well known string instead of some dotted namespace from the toml that we invent for each dependency.